.github/workflows/nightly-publish-to-npm.yml

```diff
@@ -1,12 +1,29 @@
 name: Nightly Publish Package to NPM
 on:
   schedule:
-    - cron: '0 5 * * *'
+    - cron: '0 5 * * 1-6'
+
 jobs:
   deploy-to-npm:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          persist-credentials: false

```

Comment:
I read that this makes sure the token is not stored locally. Any reason we don't want that?

Reply:
I think it is due to if we wan't this workflow to do anything we have to add cogniteCICD gihub user. i.e.

git config --global user.email "cognite-cicd@users.noreply.github.com"
git config --global user.name "Cognite CICD"

So whatever credentials it initially sets would be wrong.